### DISCUSSION TITLE
Always using word Component

### DISCUSSION THREAD

**Discussion by ilblackdragon** [near/react-on-chain#4] [Announcements]
There are a number of places in code and documentation word Widget is used.
The word "Widget" has significant degrading effect on what people feel can be built with Web Engine, and in turn limits market and confuses developers.
Component is also term from React and matches better what these elements really are.

**Comment by mpeterdev** [comment]
agreed, I will do a pass through the documentation to clean up any mentions of widget

**Reply by flowscience** [reply]
Agree. Devs can always just display 'widget' on page if the intention is to simplify end-user UX for customizable components. Docs should be formal and consistent with current nomenclature paradigms to facilitate adoption.

**Comment by lachlanglen** [comment]
100% agree. "Component" is a very familiar and clear concept to React devs and "Widget" is not.

**Comment by andy-haynes** [comment]
`<Component />` is now supported over `<Widget />`. Both will work for now but `<Widget />` will be phased out.
